weather_mv/loader_pipeline/util_test.py

```diff
@@ -178,5 +185,99 @@ def test_make_attrs_ee_compatible_a2(self):
 
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
-    # TODO(#106): Write tests...
-    pass
+
+    def test_to_json_serializable_type_none(self):
+        attrs = None
+
+        expected = None
+
+        actual = to_json_serializable_type(attrs)
+
+        self.assertEqual(actual, expected)
+        self.assertEqual(type(actual), type(expected))
+
+    def test_to_json_serializable_type_npfloat(self):
+        attrs = np.float32(1)
+
+        expected = float(1)
+
+        actual = to_json_serializable_type(attrs)
+
+        self.assertEqual(actual, expected)
+        self.assertEqual(type(actual), type(expected))
+
+    def test_to_json_serializable_type_ndarray(self):
+        attrs = np.arange(5)
+
+        expected = [0, 1, 2, 3, 4]
+
+        actual = to_json_serializable_type(attrs)
+
+        self.assertEqual(actual, expected)
```

Comment:
OK, I'll make an admission: I've written tests for these internally, but I have not yet had the time to port them to be external. I've not made these up on the fly, and I really appreciate that you're taking the initiative to fix this open issue :)

```suggestion
        self.assertIsInstance(self.convert(np.array(list(range(10)))), list)
    self.assertEqual(self.convert(np.array(list(range(10)))), list(range(10)))
    self.assertEqual(self.convert(np.array([1])), [1])
    self.assertEqual(
        self.convert(np.array([[1, 2, 3], [4, 5, 6]])), [[1, 2, 3], [4, 5, 6]]
    )
```

Reply:
I added some more I could think of. Added in https://github.com/google/weather-tools/pull/327/commits/d9cf97add436e462d01fb43fe2d32ac7c0511d17.